REQUIRE

```diff
@@ -1,7 +1,7 @@
-julia 0.6
-AWSCore 0.3
-AWSSDK 0.2.0
+julia 0.7
+AWSCore 0.5.5
+AWSSDK 0.4.0
 BSON
-Compat 0.69.0
-Memento 0.7.0
+Compat 1.5.1
```

Comment:
We don't need Compat here anymore, do we?

Reply:
ah yes you are correct. I guess the fact that I forgot a few of those compat lines was making it yell at me for not having it in.